code_samples/org-website/LockSample.js

```diff
@@ -2,10 +2,13 @@ var Client = require('hazelcast-client').Client;
 // Start the Hazelcast Client and connect to an already running Hazelcast Cluster on 127.0.0.1
 Client.newHazelcastClient().then(function (hz) {
     // Get a distributed lock called "my-distributed-lock"
-    var lock = hz.getLock("my-distributed-lock");
-    // Now create a lock and execute some guarded code.
-    return lock.lock().then(function () {
-        //do something here
+    var lock;
+    hz.getLock("my-distributed-lock").then(function (l) {
```

Comment:
can you move `// Get a distributed lock called "my-distributed-lock"` comment to just on top of this line and do the same thing for the other "org-website" examples?